languages/js/src/Host.ts

```diff
@@ -343,7 +410,13 @@ export class Host {
         );
         return { value: { Dictionary: { fields } } };
       default:
-        const instance_id = this.cacheInstance(v);
+        // TODO (dhatch): Reliable check that this is a class?
```

Comment:
Is this TODO still relevant?

Reply:
Yeah, it is. I think this is the best way to check for this